src/migrations/20190906054549-create-logins.js

```diff
@@ -1,19 +1,22 @@
 'use strict';
 module.exports = {
   up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('Users', {
+    return queryInterface.createTable('logins', {
```

Comment:
How did you reference your `logins` table to users table. No information about that contact @OlawaleJoseph about referencing logins table to users table. `userId` should be a `foreignkey` on logins table which references user table

Reply:
ok, thanks